### ISSUE TITLE
Suggestions for workshop content

### ISSUE THREAD

**Issue by martimors** [KPLauritzen/ge-workshop#1]
Let's have that discussion here (if any).

In addition to the pandas_profiling docs link, you could link to the documentation for GE's [Data Docs](https://docs.greatexpectations.io/en/latest/reference/core_concepts/data_docs.html). I don't know how it works but it looks cool, and the output is just some HTML which probably renders just fine in a notebook.

**Comment by KPLauritzen** [comment]
We could have a simple model and then do some expectations on model output. Is that what you did with Felipe, @dingobar ?

**Comment by martimors** [comment]
Yes, we did it with both input and output. For input, we used it to validate that the incoming data is valid (ie. for some error handling). For the output, we use it to test for negatives and nulls.

I do, however, think that it fits the most nicely in a model training pipeline, to document our EDA process so that the next time we train the model with that code (potentially months later) we catch anything weird.
